cmd/apps/certmanager_app.go

```diff
@@ -22,18 +22,18 @@ func MakeInstallCertManager() *cobra.Command {
 	var certManager = &cobra.Command{
 		Use:          "cert-manager",
 		Short:        "Install cert-manager",
-		Long:         "Install cert-manager for obtaining TLS certificates from LetsEncrypt",
+		Long:         "Install cert-manager for TLS certificates management",
 		Example:      "arkade install cert-manager",
 		SilenceUsage: true,
 	}
 
 	certManager.Flags().StringP("namespace", "n", "cert-manager", "The namespace to install cert-manager")
+	certManager.Flags().StringP("version", "v", "v0.15.2", "The version of cert-manager to install, has to be >=0.15.0")
```

Comment:
We may not be able to force this new version on existing users if there are any breaking changes.

Are you aware of any between 0.12.0 and this version?

Reply:
There are no breaking API changes. Upgrade path however isn't spotless due to Kubernetes limitations and moved files: https://cert-manager.io/docs/installation/upgrading/upgrading-0.13-0.14/